pkg/testutil/args.go

```diff
@@ -35,7 +35,7 @@ var argsPattern = regexp.MustCompile("`.+`")
 // surrounded by backticks with --- and then splits the resulting string by
 // spaces. Afterwards if there was a backtick matched, then we re-insert the
 // backticked content into the slice where --- is found.
-func Args(args string) []string {
+func SplitArgs(args string) []string {
```

Comment:
Drive-by review, feel free to ignore: this logic seems extraneous, straining to make this convenience available where simply using a slice literal would be simpler.

If this were my codebase, I might even go as far as having the function error when the passed string contained a quote or backtick, with the error message asking the programmer to use `[] string` to pass complex arguments.

Reply:
I found it a bit odd too, but it predates my involvement with the codebase and isn't an important thing to change (I've been focusing on making test scenarios easier to write and maintain). For all I know the scenario which needs this special handling may no longer be present!

Reply:
Fair point! Thanks for taking on this maintenance and polishing this important codebase.